src/backend/metal/src/command.rs

```diff
@@ -3237,7 +3237,15 @@ impl com::RawCommandBuffer<Backend> for CommandBuffer {
                             desc.set_clear_color(channel.interpret(raw));
                         }
                     }
-                    if op_flags.contains(native::SubpassOps::STORE) {
+                    if let Some(id) = resolve_id {
+                        let resolve = &framebuffer.attachments[id];
+                        desc.set_resolve_texture(Some(resolve));
+                        // these should only be encoded into `resolve` MTLTexture
+                        //desc.set_resolve_level();
```

Comment:
Are these still `TODO`, or could you clarify what the comment means?

Reply:
These aren't TODOs, it's something that we could be doing, but we don't need to, since our image views are already technically child MTLTexture objects that point to the right slices/levels. Do you want these comments removed?

Reply:
Oh ok, sounds good :+1: I thought that's what it meant but wasn't sure based on the comment. Maybe we could say explicitly that the image views already handle it.

Reply:
fixed now :)